api/src/main/java/org/pmiops/workbench/utils/mappers/CommonMappers.java

```diff
@@ -50,6 +50,7 @@ public static OffsetDateTime offsetDateTimeUtc(Timestamp timestamp) {
 
   public static Timestamp timestamp(OffsetDateTime offsetDateTime) {
     return Optional.ofNullable(offsetDateTime)
+        .map(odt -> odt.atZoneSameInstant(Clock.systemDefaultZone().getZone()).toOffsetDateTime())
```

Comment:
Terra times were coming in as GMT while locally we are using ET. 
This mapping ensures that all times are converted to the time zone used by AoU.